app/models/commands/todo_list/create.rb

```diff
@@ -3,6 +3,7 @@
 module Commands
   class TodoList::Create
     include Lib::Command
+    validate :unique_list_name
```

Comment:
`Lib::Command` includes rails' validation module, so we can write our own validations at this layer. One question that I have is how much the command needs to know about its event's effects in order for something to be declared 'valid.' Another: why here? The event is going to be created and its aggregate updated in the same request, so why not capture errors in the traditional modeling (aggregate) layer and bubble them up? Anyhow, this works according to my spec.